{{cookiecutter.project_slug}}/backend/poetry.lock

```diff
@@ -0,0 +1,2204 @@
+[[package]]
```

Comment:
Please remove the lock, as it could end up varying a lot depending on the system (Windows, Linux, etc). I would prefer to let users commit it if they want to, but not to have a default one there already.